spec/rubocop/cop/rspec/unspecified_exception_spec.rb

```diff
@@ -254,5 +254,22 @@ def raise_error
         }.to raise_exception(StandardError)
       RUBY
     end
+
+    it 'detects even when class definition is a direct ancestor' do
+      expect_offense(<<~RUBY)
+        expect { raise 'error' }.to class MyClass
+          raise_error
+          ^^^^^^^^^^^ Specify the exception being captured
+        end
```

Comment:
Is this even valid RSpec code?

Reply:
Oh no, these are monstrous constructions. I apologize if I undersold that in the description. 

The issue here is that to reach this condition:
```
        def find_expect_to(node)
          node.each_ancestor.find do |ancestor|
            break if ancestor.block_type?
            next unless ancestor.send_type?

            expect_to?(ancestor)
          end
        end
```

we have to put some kind of non-send, non-block node inside the `.to` statement of the expectation. 

We also only need 1 of the 2 examples I've presented here. 

So we could just go with the `if` block. Or anything else that you can think of that makes a more realistic example. 

All I am saying here is that when you type this into an editor, this describes how Rubocop will alert to it.

Reply:
Update: I was leaning into it being obvious bad code before. I got the behavior here down to using a ternary, and it is valid RSpec. My only concern is that it's slightly less obvious how problematic that is!